services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerConfig.java

```diff
@@ -438,7 +438,7 @@ public VeniceControllerConfig(VeniceProperties props) {
     this.daVinciPushStatusScanIntervalInSeconds = props.getInt(DAVINCI_PUSH_STATUS_SCAN_INTERVAL_IN_SECONDS, 30);
     this.daVinciPushStatusScanThreadNumber = props.getInt(DAVINCI_PUSH_STATUS_SCAN_THREAD_NUMBER, 4);
     this.daVinciPushStatusScanNoReportRetryMaxAttempt =
-        props.getInt(DAVINCI_PUSH_STATUS_SCAN_NO_REPORT_RETRY_MAX_ATTEMPTS, 0);
+        props.getInt(DAVINCI_PUSH_STATUS_SCAN_NO_REPORT_RETRY_MAX_ATTEMPTS, 6);
```

Comment:
Yes good catch, also found this in heap dump in controller. But why 6? Looks like a specific number to me.

Reply:
This is a random number and I was trying to increase the chance of polling DaVinci status in the worst sceanrio, and 6 * 30s = 3 mins, so at most the batch push job can be delayed by 3 mins if there is no DaVinci push status report.

Reply:
OK I see.